lib/onebox/engine/imdb_onebox.rb

```diff
@@ -0,0 +1,31 @@
+module Onebox
+  module Engine
+    class ImdbOnebox
+      include Engine
+      include StandardEmbed
+      include LayoutSupport
+
+      matches_regexp(/^https?:\/\/(?:www\.)?(?:m\.)?(?:imdb\.com)\//)
+
+      def url
+        url = @url
+        # IMDb mobile pages has blank description tag, resulting in broken onebox. Work around that.
+        url = @url.gsub("m.", "www.") if @url =~ /^http:\/\/m.imdb.com\//
```

Comment:
@eviltrout I was wondering if this should be `@url.gsub("http://m.", "http://www.")`, to be more explicit?